### DISCUSSION TITLE
Rustacean station podcast

### DISCUSSION THREAD

**Discussion by elasticdotventures** [Rust-GPU/rust-gpu.github.io#3] [Ideas]
Hi,  

I'm hoping to reach out and find somebody who might be interested in recording a https://rustacean-station.org/ podcast to chat about the rust-gpu

Cheers.

**Comment by LegNeato** [comment]
Awesome. The maintainers are discussing, we'll let you know!

**Comment by LegNeato** [comment]
@elasticdotventures We discussed, we are not quite ready. We'll loop back around once we revamp our documentation so we can better handle a potential influx of new people. Thanks!
